### ISSUE TITLE
Give pagination details for the api

### ISSUE THREAD

**Issue by wimo7083** [OperationCode/resources_api#85]
The current api is performing pagination, but the response doesn't show anything about the pagination so unless you remember it you're left scratching your head about why you only have 50 responses.

ex: 

<details>
 <summary>/api/v1/resources</summary>

```
{
  "apiVersion": "1.0", 
  "data": [
    {
      "category": "Books", 
      "downvotes": 0, 
      "id": 1, 
      "languages": [
        "multiple"
      ], 
      "name": "Free Programming Books", 
      "notes": null, 
      "paid": false, 
      "times_clicked": 0, 
      "upvotes": 0, 
      "url": "https://github.com/vhf/free-programming-books"
    }, 
    {
      "category": "Books", 
      "downvotes": 0, 
      "id": 2, 
      "languages": [
        "multiple"
      ], 
      "name": "Packtpub free programming books", 
      "notes": "Different free book daily", 
      "paid": false, 
      "times_clicked": 0, 
      "upvotes": 0, 
      "url": "https://www.packtpub.com/packt/offers/free-learning"
    }, 
    {
      "category": "Books", 
      "downvotes": 0, 
      "id": 3, 
      "languages": [], 
      "name": "Free Tech Books", 
      "notes": "Focuses on general computer science concepts rather than a specific language", 
      "paid": false, 
      "times_clicked": 0, 
      "upvotes": 0, 
      "url": "http://www.freetechbooks.com/"
    }, 
    {
      "category": "Books", 
      "downvotes": 0, 
      "id": 4, 
      "languages": [
        "multiple"
      ], 
      "name": "Programming from the Ground Up", 
      "notes": "Focuses on general computer science concepts rather than a specific language", 
      "paid": false, 
      "times_clicked": 0, 
      "upvotes": 0, 
      "url": "http://mirror.cedia.org.ec/nongnu//pgubook/ProgrammingGroundUp-1-0-booksize.pdf"
    }, 
    {
      "category": "Books", 
      "downvotes": 0, 
      "id": 5, 
      "languages": [
        "Python"
      ], 
      "name": "Teach Your Kids to Code", 
      "notes": null, 
      "paid": true, 
      "times_clicked": 0, 
      "upvotes": 0, 
      "url": "http://teachyourkidstocode.com/"
    }, 
    {
      "category": "Books", 
      "downvotes": 0, 
      "id": 6, 
      "languages": [], 
      "name": "Ed Yourdan Books", 
      "notes": "You'll have to find these books yourself to read, but they are classic CS books", 
      "paid": true, 
      "times_clicked": 0, 
      "upvotes": 0, 
      "url": "https://en.wikipedia.org/wiki/Edward_Yourdon#Publications"
    }, 
    {
      "category": "Books", 
      "downvotes": 0, 
      "id": 7, 
      "languages": [
        "multiple"
      ], 
      "name": "Safari Books Online", 
      "notes": "Free trial available", 
      "paid": true, 
      "times_clicked": 0, 
      "upvotes": 0, 
      "url": "https://www.safaribooksonline.com/our-library/#books"
    }, 
    {
      "category": "Books", 
      "downvotes": 0, 
      "id": 8, 
      "languages": [], 
      "name": "The Art of Computer Programming", 
      "notes": "General Computer Science concepts", 
      "paid": false, 
      "times_clicked": 0, 
      "upvotes": 0, 
      "url": "http://broiler.astrometry.net/~kilian/The_Art_of_Computer_Programming%20-%20Vol%201.pdf"
    }, 
    {
      "category": "Books", 
      "downvotes": 0, 
      "id": 9, 
      "languages": [], 
      "name": "The Mythical Man Month", 
      "notes": null, 
      "paid": false, 
      "times_clicked": 0, 
      "upvotes": 0, 
      "url": "https://archive.org/details/mythicalmanmonth00fred"
    }, 
    {
      "category": "Books", 
      "downvotes": 0, 
      "id": 10, 
      "languages": [], 
      "name": "Thinking Forth", 
      "notes": null, 
      "paid": false, 
      "times_clicked": 0, 
      "upvotes": 0, 
      "url": "http://thinking-forth.sourceforge.net/"
    }, 
    {
      "category": "Books", 
      "downvotes": 0, 
      "id": 11, 
      "languages": [
        "Unix/Linux/*nix"
      ], 
      "name": "The Art of Unix Programming", 
      "notes": null, 
      "paid": false, 
      "times_clicked": 0, 
      "upvotes": 0, 
      "url": "http://www.faqs.org/docs/artu/"
    }, 
    {
      "category": "Books", 
      "downvotes": 0, 
      "id": 12, 
      "languages": [], 
      "name": "The Cathedral and the Bazaar", 
      "notes": null, 
      "paid": false, 
      "times_clicked": 0, 
      "upvotes": 0, 
      "url": "http://www.catb.org/esr/writings/cathedral-bazaar/cathedral-bazaar/"
    }, 
    {
      "category": "Books", 
      "downvotes": 0, 
      "id": 13, 
      "languages": [
        "Bash"
      ], 
      "name": "Bash Beginner's Guide", 
      "notes": null, 
      "paid": false, 
      "times_clicked": 0, 
      "upvotes": 0, 
      "url": "http://www.tldp.org/LDP/Bash-Beginners-Guide/html/"
    }, 
    {
      "category": "Books", 
      "downvotes": 0, 
      "id": 14, 
      "languages": [
        "C"
      ], 
      "name": "The C Programming Language", 
      "notes": null, 
      "paid": false, 
      "times_clicked": 0, 
      "upvotes": 0, 
      "url": "https://hassanolity.files.wordpress.com/2013/11/the_c_programming_language_2.pdf"
    }, 
    {
      "category": "Books", 
      "downvotes": 0, 
      "id": 15, 
      "languages": [
        "multiple"
      ], 
      "name": "The Computer Revolution", 
      "notes": "Wiki book on everything computers", 
      "paid": false, 
      "times_clicked": 0, 
      "upvotes": 0, 
      "url": "https://en.wikibooks.org/wiki/The_Computer_Revolution"
    }, 
    {
      "category": "Books", 
      "downvotes": 0, 
      "id": 16, 
      "languages": [
        "multiple"
      ], 
      "name": "Computer Science from the Bottom Up", 
      "notes": "This book goes from operating systems fundamentals through to how applications are complied and executed", 
      "paid": false, 
      "times_clicked": 0, 
      "upvotes": 0, 
      "url": "http://feederio.com/book/33/computer-science-from-the-bottom-up"
    }, 
    {
      "category": "Books", 
      "downvotes": 0, 
      "id": 17, 
      "languages": [], 
      "name": "Zero to One", 
      "notes": "Good info for start ups", 
      "paid": true, 
      "times_clicked": 0, 
      "upvotes": 0, 
      "url": "http://buff.ly/2b7sg2E"
    }, 
    {
      "category": "Books", 
      "downvotes": 0, 
      "id": 18, 
      "languages": [
        "C"
      ], 
      "name": "Hacking The Art of Exploitation", 
      "notes": null, 
      "paid": false, 
      "times_clicked": 0, 
      "upvotes": 0, 
      "url": "https://leaksource.files.wordpress.com/2014/08/hacking-the-art-of-exploitation.pdf"
    }, 
    {
      "category": "Books", 
      "downvotes": 0, 
      "id": 19, 
      "languages": [
        "Sysadmin"
      ], 
      "name": "Debian System Administration", 
      "notes": null, 
      "paid": false, 
      "times_clicked": 0, 
      "upvotes": 0, 
      "url": "https://debian-handbook.info/browse/stable/"
    }, 
    {
      "category": "Books", 
      "downvotes": 0, 
      "id": 20, 
      "languages": [], 
      "name": "Project Gutenberg", 
      "notes": "Thousands of free e-books, some computer related, some not.", 
      "paid": false, 
      "times_clicked": 0, 
      "upvotes": 0, 
      "url": "http://www.gutenberg.org/"
    }
  ], 
  "status": "ok"
}
```
</details>

**Comment by aaron-suarez** [comment]
What should the details show? Should the json include something like 
```json
"page": 1,
"number_of_pages": 10
```
?

**Comment by wimo7083** [comment]
Yeah, maybe even the number of returned items.

**Comment by aaron-suarez** [comment]
so then
```json
"page": 1,
"number_of_pages": 10,
"records_per_page": 20
```

**Comment by Qlwentt** [comment]
I'd like to take this as my first issue. I just joined operation code. I can work on this and probably a few other tickets this weekend.

**Comment by aaron-suarez** [comment]
Ok sounds good! I'll assign it to myself so no one takes it. Hit me up on slack if you have any problems!

**Comment by aaron-suarez** [comment]
Fixed in #124
